Makefile

```
@@ -0,0 +1,123 @@
+NG_VER_TEXT     := $(shell npx -y -p @angular/cli@latest ng version 2>&1 | sed -n 's/^Angular CLI \([0-9][^ ]*\).*/\1/p' | head -n1)
+NG_VERSION      := $(if $(NG_FROM_PACKAGE),$(NG_FROM_PACKAGE),$(if $(NG_VER_JSON),$(NG_VER_JSON),$(if $(NG_VER_TEXT),$(NG_VER_TEXT),unknown)))
+
+# ----- Snapshot packaging (for AI review / clean zips) -----
+SNAP_NAME := snapshot-$(shell date +%Y%m%d-%H%M%S)
+SNAP_DIR  := .snapshots
+
+# ----- Phony targets -----
+.PHONY: help dev-backend dev-frontend dev-npm-install dev build-backend build-backend-rpi build-frontend build fmt-lint test lint fmt clean snapshot
+
+# ----- Help -----
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (15).

(maxbodylength)